debian/changelog

```diff
@@ -1,3 +1,9 @@
+gnome-initial-setup (3.38.5-1endless0) eos; urgency=medium
+
+  * Endless 3.38.0 release
```

Comment:
I think you meant:
```suggestion
  * Endless 3.38.5 release
```
Also, as a side-note, we don't need to add any manual entries to debian/changelog for packages we track on git. Jenkins will always add a new entry on top based on the latest `Version_XYZ` / `Version_XYZ_debian` tags, and then count how many commits we have from the tags to the HEAD on the respective branches.

The only exception is when we have a downstream-only epoch (ex., pulseaudio), in which case we need to add one entry just to increase the epoch. But we should never have a downstream-only epoch (bad, bad pulseaudio).

Reply:
Ouch, good catch. I always assumed bumping the Debian version in the changelog was considered a good practice, thanks for clarifying that it's not necessary though, I'll keep that in mind!